Nos/Models/ReportCategory.swift

```diff
@@ -61,7 +61,7 @@ enum ReportCategoryType {
     static let harassment = ReportCategory(
         name: .moderation.harassment,
         code: "IL-har",
-        nip56Code: .other
+        nip56Code: .profanity
```

Comment:
I think we should keep this one as `other` also. harassment != profanity and in my opinion it's better to be vague and say "other" than to be wrong. We've seen people get upset when they feel like their content has been labeled wrongly and this will only increase that.